sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -12,12 +12,73 @@
 VDM_UNFREEZE = 0
 SYSLOG_IDENTIFIER = "CCmisApi"
 
+VDM_KEY_TO_DB_KEY_PREFIX_MAP = {
+    'Modulator Bias X/I [%]' : 'biasxi',
```

Comment:
- bias name has been changed from bias_xi to biasxi
- cd name has been changed from cd_(short/long)link to cd(short/long)
- tx power/rx power names have been changed too.
Would these change cause an issue in xcvrd table parsing?

Reply:
No, this change will not cause an issue in xcvrd table parsing for TRANSCEIVER_DOM_SENSOR table.
```
                fvs = swsscommon.FieldValuePairs([(k, v) for k, v in dom_info_dict.items()])
```
https://github.com/sonic-net/sonic-platform-daemons/blob/f1511c2a5a2712d12a755c02a0db3b01c3109682/sonic-xcvrd/xcvrd/xcvrd.py#L612

Reply:
Can you add this scenario in unit test?

Reply:
@SuvarnaMeenakshi - I have modified the existing testcase to address the change in keys.